modules/scanning-images.adoc

```diff
@@ -106,13 +106,13 @@ The following vulnerability sources are not updated by the vendor:
 | `photon:1.0`^[2]^, `photon:2.0`^[2]^, `photon:3.0`^[2]^
 
 | link:https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux[{op-system-base-full}]
-| `rhel:6`^[3]^, `rhel:7`^[3]^, `rhel:8`^[3]^, `rhel:9`^[3]^
+| `rhel:6`^[3]^, `rhel:7`^[3]^, `rhel:8`^[3]^, `rhel:9`, `rhel:10`^[2]^
```

Comment:
removed the `^[3]^` because RHEL 9 came out in 2022, so the note about only working for images post June 2020 doesn't mean anything for it